src/codegen/templates/property_deleter_template.py.mako

```diff
@@ -0,0 +1,37 @@
+<%def name="script_property_deleter(attribute)">\
+<%
+        from codegen.utilities.text_wrappers import wrap
+    %>\
+    @${attribute.name}.deleter
+    def ${attribute.name}(self):
+<%
+        argtypes = []
+        for handle_parameter in attribute.handle_parameters:
+            if handle_parameter.ctypes_data_type == 'ctypes.c_char_p':
```

Comment:
look for simplified notations if possible

Reply:
I agree. You could move this logic into the `attribute.py` class to make these templates more generic.